### PR TITLE
Described to open port 54322 when doing RHV install

### PR DESCRIPTION
**Pull request by ehvs**
- For branches: 4.X
- https://bugzilla.redhat.com/show_bug.cgi?id=1974504
- Direct link to doc preview: https://deploy-preview-33726--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-default.html
- SME review: Waiting for review by @janoszen 
- Peer review: Waiting for review

### CODE REVIEW

File: modules/installation-launching-installer.adoc [hunk 1]
```diff
@@ -159,6 +159,8 @@ You can run the `create cluster` command of the installation program only once,
 
 ifndef::osp,rhv,vsphere[* Configure an account with the cloud platform that hosts your cluster.]
 
+ifdef::rhv[* Open the port `54322` in the installer machine.]
```
Comment (janoszen) [inline modules/installation-launching-installer.adoc:162]: The text should read:

> Open the port `54322` from the machine running the installer to the oVirt Engine. 

This gets rid of the ambiguity of where this port needs to be open.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ehvs** [comment]
Hi @janoszen 

Thank you! I'll apply the changes.
That said,  I got feedback in the [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1974504
) from @Gal-Zaidman too. Let me know your thoughts on that, if we may move this to **Requirements for the RHV environment** section.  And maybe we can merge both sentences?
~~~
I think that this should be documented in all installation flows.
I would move it to the "Requirements for the RHV environment" and say something like:
"Make sure that ovirt-imageio port is open on the oVirt engine (by default port 54323) and reachable from the machine the installer is running from to allow oVirt template creation by the the installer"
~~~

**Comment by janoszen** [comment]
> 
> 
> Hi @janoszen
> 
> Thank you! I'll apply the changes.
> That said, I got feedback in the [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1974504) from @Gal-Zaidman too. Let me know your thoughts on that, if we may move this to **Requirements for the RHV environment** section. And maybe we can merge both sentences?
> 
> ```
> I think that this should be documented in all installation flows.
> I would move it to the "Requirements for the RHV environment" and say something like:
> "Make sure that ovirt-imageio port is open on the oVirt engine (by default port 54323) and reachable from the machine the installer is running from to allow oVirt template creation by the the installer"
> ```

The sentence I proposed essentially says the same. I don't think we need to explain to the end user in great detail why this is needed.

**Comment by stoobie** [comment]
> The sentence I proposed essentially says the same. I don't think we need to explain to the end user in great detail why this is needed.

@janoszen Your suggestion specifies port 54322, which is the default. Gal's suggestion is more descriptive and applies to cases in which someone might have specified a different port number.

@ehvs I suggest:

> Open the ovirt-imageio port from the machine running the installer to the oVirt Engine. By default, this is port 54322.

**Comment by Gal-Zaidman** [comment]
> the ovirt-imageio port from the machine running the installer to the oVirt Engine. By default, this is port 54322

I'm ok with that

**Comment by ehvs** [comment]
@stoobie  @Gal-Zaidman @janoszen 
Updated to reflect the suggestions. Let me know is ok.

File: modules/installation-launching-installer.adoc [hunk 2]
```diff
@@ -159,6 +159,8 @@ You can run the `create cluster` command of the installation program only once,
 
 ifndef::osp,rhv,vsphere[* Configure an account with the cloud platform that hosts your cluster.]
 
+ifdef::rhv[* Open the `ovirt-imageio` port from the machine running the installer to the oVirt Engine. By default, port `54322`.]
```
Comment (mikemckiernan) [inline modules/installation-launching-installer.adoc:162]: nitpicky stuff...
* Do you mind changing to "...the machine that is running the installer..."?
* I asked in Slack if we can locate the customer-facing name for "oVirt Engine."
* Do you mind using a complete sentence at the end? "By default, the port is `...`."
* Make sure 54322 is the port that you want.  I skimmed the BZ and saw 54323 mentioned.  FYI, I have no technical experience in this area.

Comment (stoobie) [reply modules/installation-launching-installer.adoc:162]: @mikemckiernan oVirt Engine is upstream term. I actually opened a new bug about this. In RHV we use an attribute as a stand-in for this term. In the upstream it evaluates to "Engine" and in the downstream it evaluates to "Manager".

There is currently no such attribute in the OCP attributes file, but I would like to add one. For now, @ehvs I suggest changing "the oVirt Engine" to "the engine".

**Comment by ehvs** [comment]
@stoobie @mikemckiernan 
Both of the suggestions were applied, lmk your thoughts.

File: modules/installation-launching-installer.adoc [hunk 3]
```diff
@@ -159,6 +159,8 @@ You can run the `create cluster` command of the installation program only once,
 
 ifndef::osp,rhv,vsphere[* Configure an account with the cloud platform that hosts your cluster.]
 
+ifdef::rhv[* Open the `ovirt-imageio` port from the machine running the installer to the engine. By default, the port is `54322`.]
```
Comment (stoobie) [inline modules/installation-launching-installer.adoc:162]:
```suggestion
ifdef::rhv[* Open the `ovirt-imageio` port to the Engine from the machine running the installer. By default, the port is `54322`.]
```
@ehvs I double-checked and it should be "Engine" with a capital "E". Sorry I didn't notice that before. And I suggest moving the clause "to the Engine" earlier in the sentence to make easier to parse.

**Review by stoobie — CHANGES_REQUESTED** [review]
Just two minor requests.

**Comment by ehvs** [comment]
Hi @stoobie 
Changes done. Lmk :)

File: modules/installation-launching-installer.adoc [hunk 4]
```diff
@@ -159,6 +159,8 @@ You can run the `create cluster` command of the installation program only once,
 
 ifndef::osp,rhv,vsphere[* Configure an account with the cloud platform that hosts your cluster.]
 
+ifdef::rhv[* Open the `ovirt-imageio` port to the Engine from the machine running installer. By default, the port is `54322`.]
```
Comment (stoobie) [inline modules/installation-launching-installer.adoc:162]:
```suggestion
ifdef::rhv[* Open the `ovirt-imageio` port to the Engine from the machine running the installer. By default, the port is `54322`.]
```

**Review by stoobie — CHANGES_REQUESTED** [review]
Hi @ehvs  You accidentally dropped "the" before "installer". Other than that, LGTM!

**Comment by ehvs** [comment]
@stoobie  great catch! :D just fixed it.
@mikemckiernan  @janoszen  @Gal-Zaidman  lmk your final thoughts if we can push this.

**Comment by mikemckiernan** [comment]
@ehvs, we're in a merge freeze period until 4.8 is posted.  We can merge this after 4.8 GA, which I believe is 12JUL.

**Pull request merged**